nodes/src/nodes/milvus/milvus.py

```
@@ -480,13 +512,39 @@ def remove(self, objectIds: List[str]) -> None:
         filter_expression = ' and '.join(must_conditions) if must_conditions else None
         if filter_expression:
-            self.client.delete(collection_name=self.collection, filter=filter_expression)
+            try:
+                self.client.delete(collection_name=self.collection, filter=filter_expression, timeout=self.timeout)
+            except Exception as e:
+                engLib.debug(f'Error removing documents: {e}')
+                raise
 
         return
 
+    def _batchUpsertResults(self, results: List[dict], isDeleted: bool) -> None:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)